src/script/ismine.cpp

```diff
@@ -142,6 +142,9 @@ isminetype IsMine(const CKeyStore &keystore, const CScript& scriptPubKey, bool&
             return ISMINE_SPENDABLE;
         break;
     }
+
+    case TX_WITNESS_UNKNOWN:
```

Comment:
I think this should be above with the `TX_NULL_DATA` and `TX_NONSTANDARD` cases.

Reply:
Nice catch, done.